packages/mobile/src/components/core/Tile.tsx

```diff
@@ -71,24 +74,52 @@ export const Tile = <
     ...other
   } = props
 
+  const {
+    scale,
+    handlePressIn: handlePressInScale,
+    handlePressOut: handlePressOutScale
+  } = usePressScaleAnimation()
```

Comment:
Probably good to pass the `scale` prop all the way through from a Tile prop so that callers can customize. I'm sure we'll want to tweak it based on the size of some objects at some point in time